third-party/libtm/fw/CMakeLists.txt

```diff
@@ -2,8 +2,8 @@
 # Copyright(c) 2019 Intel Corporation. All Rights Reserved.
 cmake_minimum_required(VERSION 3.1.3)
 
-set( FW_VERSION "0.0.18.5715")
-set( FW_SHA1 cc12cf05f387d80f65e98c4d41fc883725124a08)
+set( FW_VERSION "0.0.18.6100")
+set( FW_SHA1 858b786215ff66cf8cf93fa85393ca6268dedee4)
```

Comment:
@radfordi , the FW 6100 has already been uploaded. If this PR was pending the FW release - then pls check and confirm to merge.

Reply:
@ev-mp, I confirmed that the FW had been uploaded before pushing this particular commit.